robot_smach_states/examples/simple_navigate.py

```diff
@@ -22,26 +22,31 @@
         sys.exit()
 
     robot = Robot()
-    
+
     if len(sys.argv) > 2:
         room = sys.argv[2]
     else:
         room = "kitchen"
 
     rospy.sleep(2) # wait for tf cache to be filled
-    
+
     #nav_state = NavigateToPose(robot, 1, 0, 0)
     #nav_state.execute()
 
     #nav_state = NavigateToObserve(robot, ds.EntityByIdDesignator(robot=robot, id="dinnertable"))
     #nav_state.execute()
 
-    nav_state = NavigateToSymbolic(robot, {
-                                            ds.EntityByIdDesignator(robot, id=room):"in"},
-                                            ds.EntityByIdDesignator(robot, id=room))
-    nav_state.execute()
-    
+    # nav_state = NavigateToSymbolic(robot, {
+    #                                         ds.EntityByIdDesignator(robot, id=room):"in"},
+    #                                         ds.EntityByIdDesignator(robot, id=room))
+    # nav_state.execute()
+
     #nav_state = NavigateToSymbolic(robot, {
     #                                        ds.EntityByIdDesignator(robot, id="couch"):"in_front_of"},
     #                                        ds.EntityByIdDesignator(robot, id="couch"))
     #nav_state.execute()
+
+    nav_state = NavigateToWaypoint(robot=robot, waypoint_designator=ds.EntityByIdDesignator(robot=robot,
+                                                                                         id="hsr_demo_waypoint"),
```

Comment:
why hsr_demo_waypoint??? Plus, why the change from NavigateToSymbolic to NavigateToWaypoint?

Reply:
This is just an example, why care ;)